RELEASE.md

```diff
@@ -0,0 +1,39 @@
+# Strimzi Kafka OAuth release process
+
+This document describes the steps needed for release of Strimti Kafka OAuth library.
+
+## Create new release branch
+
+The release branch should be named `release-M.N.x` where `M.N` is replaced with the major and minor versions of the new release.
+For example:
+
+```
+git checkout -b release-0.2.x
+```
+
+## Set new version in main Maven project
+
+Update the version for the new Maven project:
+
+```
+mvn versions:set -DnewVersion=X.Y.Z
+```
+
+Where `X.Y.Z` is the new version which is going to be released.
+
+## Update dependencies in examples and tests
+
+Update the version in dependencies in:
```

Comment:
I propose more explicit directions:

Update the `strimzi-oauth.version` property in: